simplematrixbotlib/match.py

```diff
@@ -83,7 +85,27 @@ def __init__(self, room, event, bot, prefix="") -> None:
         super().__init__(room, event, bot)
         self._prefix = prefix
 
-    def command(self, command=None):
+        """Forms of identification"""
+        self._own_user_id = room.own_user_id
+        self._own_nio_user = self.room.users[self._own_user_id]
+        self._own_disambiguated_name = self._own_nio_user.disambiguated_name
+        self._own_display_name = self._own_nio_user.display_name
+        self._own_display_name_colon = f"{self._own_display_name}:"
+        self._own_pill = f"<a href=\"https://matrix.to/#/{self._own_user_id}\">"
+
+        self.mention() # Set self._mention_id_length
+        self._body_without_prefix = self.event.body[len(self._prefix):]
+        self._body_without_mention = self.event.body[self._mention_id_length:]
+        
+        if self.mention():
+            body = self._body_without_mention
+        elif self.prefix():
+            body = self._body_without_prefix
+        else:
+            body = self.event.body
```

Comment:
why use distinct member variables? they can't both apply at once

Reply:
Can you reword this?